packages/database/dev/src/lib/migrations.test.ts

```
@@ -173,7 +177,7 @@ test('Throws MigrationFileNotFoundError for directory missing migration.sql', as
   db = await PGlite.create()
 
   await expect(applyMigrations(db, tmpDir.path, '0001_missing_file')).rejects.toThrow(
-    'migration.sql not found in migration directory',
+    /Migration SQL file not found: .*0001_missing_file[\/\\]migration.sql/,
```

ESLint

[error] Unnecessary escape character: \/.

(no-useless-escape)